lib/txt/motd

```diff
@@ -1,24 +1,23 @@
      <Y><z>Important game status information:<z>
 
-      <R>Sneezy Restore Status:<z> Previous player files are not available 
-      (pre-sneezy.com). View news for upate on player merge with UNDO.IT. 
-      For returing players we will restore/load up to 5 items excluding 
-      artifacts and +50 level gear. Custom items obtained by quests
-      or events made for players will be returned. Players with homes 
-      will be provided keys to their homes.<z>
+      <R>Offical Status:<z> PWIPE 07/26/2018
+      Limit 1 Account. 2 simultaneous character logons per account. 
+      Multiclassing Re-Enabled. Access dual-classing at lvl 50.  
+      Players with homes will be provided keys to their homes upon request.<z>
 
 
      <Y>SNEEZYMUD CONTACT:<z>       <c>support@sneezymud.com<z>
+     <Y>DISCORD CHANNEL:<z>       <c>https://discord.gg/YY2PgqP<z>
```

Comment:
Delete the email? Or add a working one, perhaps, but having a broken email is much worse than having no email at all.